tests/sentry/api/serializers/test_project.py

```diff
@@ -690,6 +690,7 @@ def test_truncated_latest_release(self):
         assert "releases" in result["features"]
         assert result["platform"] == self.project.platform
         assert result["latestRelease"] == {"version": self.release.version}
+        assert "sentry:token" not in result["options"]
```

Comment:
can you also test `sentry:symbol_sources`, which was the key our team was specifically interested in.